src/overlays/actors/ovl_En_Po_Fusen/z_en_po_fusen.c

```diff
@@ -143,20 +145,22 @@ void EnPoFusen_Destroy(Actor* thisx, PlayState* play) {
     Collider_DestroySphere(play, &this->collider);
 }
 
+/**
+ * Search for Romani's actor, beacuse it's PoFusen's job to update her actor on pop.
+ */
 u16 EnPoFusen_CheckParent(EnPoFusen* this, PlayState* play) {
-    Actor* actorPtr;
+    Actor* actorSearchPtr = play->actorCtx.actorLists[ACTORCAT_NPC].first;
 
-    actorPtr = play->actorCtx.actorLists[ACTORCAT_NPC].first;
     if (ENPOFUSEN_IS_FUSE_TYPE(&this->actor)) {
         return true;
     }
 
-    while (actorPtr != NULL) {
-        if (actorPtr->id == ACTOR_EN_MA4) {
-            this->actor.parent = actorPtr;
+    while (actorSearchPtr != NULL) {
+        if (actorSearchPtr->id == ACTOR_EN_MA4) {
+            this->actor.parent = actorSearchPtr;
             return true;
         }
-        actorPtr = actorPtr->next;
+        actorSearchPtr = actorSearchPtr->next;
```

Comment:
I think nearly all actor variables are ptrs, so I don't think that prefix is needed. But `actorSearch` alone doesn't work too well? Some uses in the repo are `npc`, `npcActor`, `actor`, `actorIter`, `actorIteration` i.e. it's not consistent at all so I'll leave it to your judgement